LiskHQ/lisk-sdk
# Update commander network identifier - Closes #4336

### What was the problem?
- transaction:create
- transaction:verify
- transaction:sign
- signature:create
Needed network identifier to be added.

### How did I solve it?
Add `networkIdentifier` as a flag (it can specify, main | test or custom hash), if the setting does not exist, it uses `config.api.network` value (main or test).
If that doesn't exist, it will throw an error

### How to manually test it?
Create a transaction using those command, and send it to the node and observe it will be accepted.

### Review checklist

- [ ] The PR resolves #4336
